tests/test_api.py

```diff
@@ -35,13 +35,9 @@
         ("eccc", "observation", {"parameter": "daily", "resolution": "daily"}, None),  # noqa: E800, ERA001
         # IMGW Hydrology
         # FIXME: Deactivated after flaw in data discovered on 2023-09-26
```

Comment:
This is a leftover remark about the act of deactivating those tests. It will do no harm, but it can be removed ;].